client/client.go

```diff
@@ -30,24 +32,21 @@ func (e StatusError) Error() string {
 // Client represents a HTTP client for making requests to the FullStory API.
 type Client struct {
 	HTTPClient *http.Client
-	Config
-}
-
-// Config is configuration for Client.
-type Config struct {
-	APIToken string
-	BaseURL  string
+	Config     *config.Config
+	BaseURL    string
 }
 
 // NewClient returns a Client initialized with http.DefaultClient and the
 // supplied apiToken.
-func NewClient(apiToken string) *Client {
+func NewClient(config *config.Config) *Client {
```

Comment:
Just a strict move from an external repo. The only change is referencing the config directly rather than an intermediate object.